pinery-lims-flatfile/src/main/java/ca/on/oicr/pinery/lims/flatfile/dao/SampleFileDao.java

```diff
@@ -64,6 +64,7 @@ public SampleProject mapRow(ResultSet rs, int rowNum) throws SQLException {
       p.setArchivedCount(ModelUtils.nullIfZero(rs.getInt("archivedCount")));
       p.setEarliest(ModelUtils.convertToDate(rs.getString("earliest")));
       p.setLatest(ModelUtils.convertToDate(rs.getString("latest")));
+      p.setActive(true);
```

Comment:
Should actually be written to and read from flat file

Reply:
How? The project information is synthesised from the sample information. Should each file get an active flag that gets aggregated?

Reply:
The project data should probably be written to its own file. As is, the attribute will always be true in Pinery deployments serving flat files, which should be our main data sources for production.

Reply:
Okay. Fixed.